lib/AST/Decl.cpp

```diff
@@ -5599,6 +5599,53 @@ bool VarDecl::isPropertyMemberwiseInitializedWithWrappedType() const {
       this, /*checkDefaultInit=*/false);
 }
 
+static bool isInitFirstParamEscapingClosure(const ConstructorDecl *init) {
+  Type initTy = init->getInterfaceType();
+  if (!initTy || !initTy->is<AnyFunctionType>())
+    return false;
+
+  Type resultTy = initTy->castTo<AnyFunctionType>()->getResult();
+  if (!resultTy || !resultTy->is<FunctionType>())
```

Comment:
Similarly, you can assume that structural components of types are not null.

Reply:
Right, okay.